components/resources/library/src/androidMain/AndroidManifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application>
+        <provider
+                android:authorities="org.jetbrains.compose.resources.AndroidContextProvider"
```

Comment:
```suggestion
                android:authorities="${applicationId}.resources.AndroidContextProvider"
```
The site says:
```
If we would use a fixed authorities string, we would only be able to run on one Application per Android Device
...
This is why we have to double check if the applicationId is NOT the one we’re using
```

I tried to change `applicationId`, and couldn't install it because of `INSTALL_FAILED_CONFLICTING_PROVIDER`